<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Hanuma" failures="2" tests="7" name="Test" time="51.336" errors="0" timestamp="2024-08-02T10:51:19 IST">
  <testcase classname="testCases.TC02_LoginTest" name="verify_login" time="1.989"/>
  <testcase classname="testCases.TC01_AccountRegistration" name="testRegistration" time="2.634"/>
  <testcase classname="testCases.TC03_LoginDataDrivenTest" name="verify_LoginDDT" time="1.852"/>
  <testcase classname="testCases.TC03_LoginDataDrivenTest" name="verify_LoginDDT" time="11.221"/>
  <testcase classname="testCases.TC03_LoginDataDrivenTest" name="verify_LoginDDT" time="11.099">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC03_LoginDataDrivenTest.verify_LoginDDT(TC03_LoginDataDrivenTest.java:43)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_LoginDDT -->
  <testcase classname="testCases.TC03_LoginDataDrivenTest" name="verify_LoginDDT" time="11.234"/>
  <testcase classname="testCases.TC03_LoginDataDrivenTest" name="verify_LoginDDT" time="1.963">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC03_LoginDataDrivenTest.verify_LoginDDT(TC03_LoginDataDrivenTest.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_LoginDDT -->
</testsuite> <!-- Test -->
